SamplesDashboard/SamplesDashboard/ClientApp/src/views/repositories/Repositories.tsx

```diff
@@ -143,21 +142,22 @@ IRepositoryState> {
         let majorUpdateCount = 0;
         let urgentUpdateCount = 0;
         for (const item of items) {
-            if (item.vulnerabilityAlerts.totalCount > 0) {
-                urgentUpdateCount = urgentUpdateCount + 1;
-            } else {
+            if (item.vulnerabilityAlerts && item.vulnerabilityAlerts && item.vulnerabilityAlerts.totalCount > 0) {
+                urgentUpdateCount++;
+            }
+            else {
                 switch (item.repositoryStatus) {
                     case 1:
```

Comment:
It is not clear what the case 1/2/3 values are. This can change to an enum so that it is clear what the cases are checking for

Reply:
We store the status values as an enum in the backend, then we send them to the front end.